samples/quickstart.js

```diff
@@ -15,56 +15,75 @@
 
 'use strict';
 
-async function main(
-  projectId,
-  location,
-  gcsInputUri = 'gs://cloud-samples-data/documentai/invoice.pdf'
-) {
+async function main(projectId, location, processorId, filePath) {
   // [START documentai_quickstart]
   /**
    * TODO(developer): Uncomment these variables before running the sample.
    */
   // const projectId = 'YOUR_PROJECT_ID';
   // const location = 'YOUR_PROJECT_LOCATION'; // Format is 'us' or 'eu'
-  // const gcsInputUri = 'YOUR_SOURCE_PDF';
+  // const processor = 'YOUR_PROCESSOR_ID'; // Create processor in Cloud Console
+  // const filePath = '/path/to/local/pdf';
 
   const {
-    DocumentUnderstandingServiceClient,
-  } = require('@google-cloud/documentai');
-  const client = new DocumentUnderstandingServiceClient();
+    DocumentProcessorServiceClient,
+  } = require('@google-cloud/documentai').v1beta3;
+
+  const clientOptions = {
+    apiEndpoint: 'us-documentai.googleapis.com',
+  };
+
+  const client = new DocumentProcessorServiceClient(clientOptions);
 
   async function quickstart() {
-    // Configure the request for processing the PDF
-    const parent = `projects/${projectId}/locations/${location}`;
+    // The full resource name of the processor, e.g.:
+    // projects/project-id/locations/location/processor/processor-id
+    // You must create new processors in the Cloud Console first
+    const name = `projects/${projectId}/locations/${location}/processors/${processorId}`;
+
+    // Read the file into memory.
+    const fs = require('fs');
+    const imageFile = fs.readFileSync(filePath);
```

Comment:
Same feedback for sync methods here and anywhere else

Reply:
Done.